app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -27,20 +27,76 @@
               @change="statusFields = true"
             />
 
+            <div v-show="statusFields">
+              <div v-if="showRevising">
+                <label>Submitter's Email</label>
+                <v-autocomplete
+                  v-model="assignee"
+                  clearable
+                  dense
+                  :filter="autoCompleteFilter"
+                  :items="formReviewers"
+                  :loading="loading"
+                  no-data-text="No Form Reviewers found with search. Add Form Reviewers on the Manage page."
+                  outlined
+                  return-object
+                  :rules="[(v) => !!v || 'Assignee is required']"
+                >
+                  <!-- selected user -->
+                  <template #selection="data">
+                    <span
+                      v-bind="data.attrs"
+                      :input-value="data.selected"
+                      close
+                      @click="data.select"
+                      @click:close="remove(data.item)"
+                    >{{ data.item.fullName }}</span>
+                  </template>
+                  <!-- users found in dropdown -->
+                  <template #item="data">
+                    <template v-if="typeof data.item !== 'object'">
+                      <v-list-item-content v-text="data.item" />
+                    </template>
+                    <template v-else>
+                      <v-list-item-content>
+                        <v-list-item-title v-html="data.item.fullName" />
+                        <v-list-item-subtitle v-html="data.item.username" />
+                        <v-list-item-subtitle v-html="data.item.email" />
+                      </v-list-item-content>
+                    </template>
+                  </template>
+                </v-autocomplete>
+                <span v-if="assignee">Email: {{ assignee.email }}</span>
+                <v-checkbox v-model="addComment" :label="'Add Comment to Email'"></v-checkbox>
```

Comment:
maybe this should be 'Add comment to notification' or 'Include a comment in the email notification the submitter will receive'